lib/tasks/pulmap.rake

```diff
@@ -89,6 +89,20 @@ namespace :pulmap do
     end
   end
 
+  namespace :delete_record do
+    desc "Delete a single record"
+    task :index, :id do |_t, args|
+      # Set up connection and args
+      solr = Blacklight.default_index.connection
```

Comment:
suggestion: Move line 96 below 98, so that we check if there is an id param before setting up the solr connection.